tools/src/zopensetupmain.c

```diff
@@ -43,11 +43,22 @@ static void syntax(const char* pgm) {
                   "    sub-directories created for each of the tools needed for running the zopen utility\n"
                   "Options:\n"
                   " -v : print out verbose messages\n"
-                  " -q : only print out errors\n",
+                  " -q : only print out errors\n"
+                  "Note:\n"
+                  " Special consideration is made if <root> is ${HOME}/zopen in which case no link is created\n",
                   pgm, ZOPEN_TOOLS_URL, pgm);
   return;
 }
 
+#define _CVTSTATE_OFF     0
```

Comment:
Can you include _Nascii.h instead or it only works with -qASCII?

Reply:
it only works _without_ -qASCII... I used the macros just because that's what was in the book. Wasn't sure if _Nascii.h was an external header we could use?